docs/contributing.md

```diff
@@ -33,21 +33,81 @@ The next Community Call will be held on October 24th, 2024 at [5:00 PM CEST](htt
 Please use the [project's issue tracker](https://github.com/headlamp-k8s/headlamp/issues) for filing any bugs you find or features
 you think are useful.
 
+### Guidelines for Submitting an Issue
+
+When submitting an issue, follow these guidelines to help maintainers address it efficiently:
+
+- **Search for Existing Issues:** Use the issue tracker to see if your issue already exists. If it does, consider adding your input to the existing issue instead of opening a new one.
+- **Structure Your Issue Description:**  
+  - **Summary:** Briefly explain the issue.  
+  - **Expected Behavior:** What did you expect to happen?  
+  - **Observed Behavior:** What actually happened?  
+  - **Steps to Reproduce:** List clear steps to reproduce the issue.  
+- **Add Relevant Labels:** Use labels such as `bug`, `desktop` to categorize your issue properly.
```

Comment:
We now have issue templates, so we shouldn't have a duplicate guide here. I'd suggest:
1. search for whether the issue/feature has been filed already
2. file the issue template depending on whether it's a bug/feature

Reply:
done!